src/python/pants/init/engine_initializer.py

```diff
@@ -240,14 +242,14 @@ def run_goal_rules(
     return PANTS_SUCCEEDED_EXIT_CODE
 
   def create_build_graph(
-    self, target_roots: TargetRoots, build_root: Optional[str] = None,
+    self, parsed_specs: ParsedSpecs, build_root: Optional[str] = None,
   ) -> Tuple[LegacyBuildGraph, LegacyAddressMapper]:
     """Construct and return a `BuildGraph` given a set of input specs."""
-    logger.debug('target_roots are: %r', target_roots)
+    logger.debug('parsed_specs are: %r', parsed_specs)
```

Comment:
Should we use f-string syntax here?

Reply:
No, as mentioned over DM. See https://stackoverflow.com/questions/13500813/how-to-use-modern-string-formatting-options-with-pythons-logging-module.